docs/development/release.md

```diff
@@ -72,59 +124,66 @@ Similar to #4, make sure the DATA_MODEL_CHANGELOG is updated if there are data m
 git add CONTRIBUTORS.md
 ```
 
+Commit and push these changes, if any.
+
 ### 6. Update Cumulus package API documentation
 
 Update auto-generated API documentation for any Cumulus packages that have it:
 
 ```bash
-  npm run docs-build-packages
+npm run docs-build-packages
 ```
 
+Commit and push these changes, if any.
+
 ### 7. Cut new version of Cumulus Documentation
 
-If this is a backport, do not version documentation. For various reasons, we do not merge backports back to master, other than changelog notes. Doc changes will not be published to our documentation website.
+> Note: This is for version `1.10.3` or later.
+
+If this is a backport, do not create a new version of the documentation. For various reasons, we do not merge backports back to master, other than changelog notes. Documentation changes for backports will not be published to our documentation website.
 
-```shell
+```bash
 cd website
 npm run version ${release_version}
 git add .
 ```
 
 Where `${release_version}` corresponds to the version tag `v1.2.3`, for example.
 
-Note: This is for 1.10.3 or later.
+Commit and push these changes.
 
 ### 8. Create a pull request against the minor version branch
 
 1. Push the release branch (e.g. `release-1.2.3`) to GitHub.
-2. Create a PR against the minor version base branch  e.g. `release-1.2.x`).
-3. Configure CI to run automated tests against this PR by finding the branch plan for the release branch and setting these variables:
+2. Create a PR against the minor version base branch (e.g. `release-1.2.x`).
+3. Configure CI to run automated tests against this PR by finding the branch plan for the release branch (`release-1.2.3`) and setting these variables:
```

Comment:
Do we set these variables in Bamboo?

Reply:
Yeah, I can update that to be more clear

Reply:
35d6eb91364027eb7e5441a4823817672a3d8223